src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -195,23 +196,28 @@ static ucs_status_t
 uct_cuda_ipc_mem_reg_internal(uct_md_h uct_md, void *addr, size_t length,
                               unsigned flags, uct_cuda_ipc_key_t *key)
 {
+    ucs_log_level_t log_level;
     CUdevice cu_device;
     ucs_status_t status;
 
     if (!length) {
         return UCS_OK;
     }
 
-    status = UCT_CUDADRV_FUNC(cuIpcGetMemHandle(&(key->ph), (CUdeviceptr) addr));
+    log_level = (flags & UCT_MD_MEM_FLAG_HIDE_ERRORS) ? UCS_LOG_LEVEL_DEBUG :
```

Comment:
maybe declare `log_level` with `const` qualifier and assign the value during initialization, wdyt?
otherwise - align by `=` with the statement below